src/Compute/Compute.Test/ScenarioTests/VirtualMachineTests.ps1

```diff
@@ -7441,3 +7441,50 @@ function Test-VMTLWithGallerySourceImage
         Clean-ResourceGroup $rgname;
     }
 }
+
+
+<#
+.SYNOPSIS
+Windows machine test ensuring the EnableAutoUpdate value on the 
+provided VM is not overwritten. 
+#>
+function Test-CapacityReservationSharingProfile
+{
+    # Setup
+    $rgname = Get-ComputeTestResourceName;
+    $loc = 'WestEurope';
```

Comment:
use the method that gets a location automatically. Any reason to hardcode WestEurope?

Reply:
feature is available in limited locations at the moment behind an AFEC flag